README.md

```diff
@@ -1 +1,48 @@
-# TodoList
\ No newline at end of file
+# TodoList
+This is a project about to-do's . Projects can be added and tasks(to-do's) added under each project.  The priorities chosen will display a different background color for any todo(task)  The to-do's and projects will be stored on local storage.
+ 
+![screenshot](./dist/assets/images/todoshotOne.png)
+
+## Built With
+
+- HTML & CSS
+- BOOTSTRAP FRAMEWORK
+- Javasript
+
+## Getting started
+   To get a local copy up and running follow these simple example steps.
+# Setup
+- Clone the project locally.
+- Type the command `cd TodoList`
+- run the command   `yarn install`
+- Use any browser of your choice, copy the absolute path of the  `index.html` and type in the search bar
+## Live Demo
+
+- [Live Demo Link](https://che30.github.io/TodoList/)
+ 
+
+## Author
+**Che Blanchard**
+
+- GitHub: [@che30](https://github.com/che30)
+- LinkedIn: [Che Blanchard](https://www.linkedin.com/in/che-nsoh-9455271b0/)
+- Twitter:[che55085128](https://twitter.com/che55085128)
+
+
+## Acknowledgements
+- The next web
+- W3 schools
+- Font awesome
+- Stack overflow
+
+##  Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## License
+
+This project is [MIT](./LICENSE.txt) licensed.
```

Comment:
## Fix or remove dead links.
- [ ] the license link is currently a dead link, please fix this by either adding a working link to the project's licence or remove the link for a much cleaner README, thank you.